packages/data-api/scripts/createBuildAnalyticsTableFn.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -e
+
+DIR=$(pwd "$0")
+source $DIR/../../scripts/bash/mergeCurrentEnvWithEnvFile.sh 
+
+# Set default port in case it wasn't in .env
+: "${DB_PORT:=5432}"
+
+cd scripts
+export PGPASSWORD=$DB_PASSWORD
+psql -p $DB_PORT -h $DB_URL -d $DB_NAME -U $DB_USER < createBuildAnalyticsTableFn.sql
```

Comment:
Not too opinionated, but `psql` also has the `-f` flag for passing in a file if you prefer that?

Reply:
Actually I prefer pipes :) Generic shell syntax